backend/src/main/java/com/happy/friendogly/playground/service/PlaygroundCommandService.java

```diff
@@ -137,4 +153,20 @@ public UpdatePlaygroundMemberMessageResponse updateMemberMessage(
         playgroundMember.updateMessage(request.message());
         return new UpdatePlaygroundMemberMessageResponse(playgroundMember.getMessage());
     }
+
+    @Scheduled(cron = EVERY_HOUR)
+    public void deleteJoinMemberIntervalTime() {
+        LocalDateTime twoHoursAgo = LocalDateTime.now().minusHours(2);
```

Comment:
미리 정의해둔 상수를 사용해도 될 듯 합니다~
```suggestion
        LocalDateTime twoHoursAgo = LocalDateTime.now().minusHours(OUTSIDE_MEMBER_KEEP_TIME);
```